python/lsst/obs/base/mapper_tests.py

```diff
@@ -170,8 +170,14 @@ def _test_map(self, butlerLocation, dataId):
 
     def test_map(self):
         dataId = self.dataIds['raw']
-        self._test_map(self.mapper.map_raw(dataId), dataId)
-        self._test_map(self.mapper.map("raw", dataId), dataId)
+        location = self.mapper.map_raw(dataId)
+        if not isinstance(location, lsst.daf.persistence.butlerLocation.ButlerComposite):
+            self._test_map(location, dataId)
+        else:
+            self.log.info('ButlerComposite datasets are not tested for mapper functions')
```

Comment:
Maybe `log.warn`? At least have some more information as to why they aren't tested.

Reply:
Warn does seem appropriate.